shared/health/src/test/java/io/pravega/shared/health/HealthManagerTests.java

```diff
@@ -80,14 +79,14 @@ public void testHealth() throws Exception {
      */
     @Test
     public void testHealthInvalidName() {
-        Assert.assertNull("An exception should be thrown given an unregistered contributor.",
-                service.getEndpoint().getHealth("unknown-contributor-name"));
+        AssertExtensions.assertThrows("An exception should be thrown given an unregistered contributor.",
+                () -> TestHealthContributors.awaitHealthContributor(service, "unknown-contributor-name"),
+                e -> e instanceof TimeoutException);
```

Comment:
What is the timeout here? I hope it is really small, otherwise I would prefer if this test didn't take too long.

Is this test taking less than 1 second to finish?

Reply:
It times out after 3 times the interval supplied for the `HealthServiceManager` which is 1 second in this case. I will change all instances to instead take a timeout of 100 milliseconds.